src/components/Avatar.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+  import { t } from 'svelte-i18n';
+  import LogoGitHub32 from 'carbon-icons-svelte/lib/LogoGithub32';
+  import LogoLinkedIn32 from 'carbon-icons-svelte/lib/LogoLinkedin32';
+  import LogoTwitter32 from 'carbon-icons-svelte/lib/LogoTwitter32';
+  import Earth32 from 'carbon-icons-svelte/lib/Earth32';
+
+  // eslint-disable-next-line
```

Comment:
Remove this comment and update the eslint configuration with `"imports/prefer-default-export": "off"`